flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -603,6 +655,10 @@ private Operation convertSqlQuery(SqlNode node) {
 	 * @return TableSchema
 	 */
 	private TableSchema createTableSchema(SqlCreateTable sqlCreateTable) {
+		// Unique key and enforced mode are not supported yet.
+		sqlCreateTable.getFullConstraints()
+				.forEach(this::validateTableConstraint);
```

Comment:
Could we move this validation to the beginning of `convertCreateTable` to make it more visible to developers?